### ISSUE TITLE
Miden package support in cargo extension

### ISSUE THREAD

**Issue by greenhat** [0xPolygonMiden/compiler#145]
### Goal 
Generate Miden package as a target artifact.
Use Miden packages as dependencies.

**Comment by bitwalker** [comment]
@greenhat I think this is complete, no?

**Comment by greenhat** [comment]
Yes, we still have to properly implement Miden package as a dependency. We have https://github.com/0xPolygonMiden/compiler/issues/368 for it, so this issue can be closed.
